src/app/news/news-list.component.ts

```diff
@@ -20,6 +20,7 @@ import { DialogsPromptComponent } from '../shared/dialogs/dialogs-prompt.compone
 export class NewsListComponent {
 
   @Input() items: any[] = [];
+  @Input() editSuccessMessage: string = 'News has been updated successfully.';
```

Comment:
Codeclimate states not to declare type when value is assigned and value is self declarable.